easybuild/framework/easyconfig/default.py

```diff
@@ -35,37 +35,35 @@
 """
 from vsc.utils import fancylogger
 
-from easybuild.tools.deprecated.eb_2_0 import ExtraOptionsDeprecatedReturnValue
-from easybuild.tools.ordereddict import OrderedDict
 
 _log = fancylogger.getLogger('easyconfig.default', fname=False)
 
 
 # we use a tuple here so we can sort them based on the numbers
-HIDDEN = "HIDDEN"
-MANDATORY = "MANDATORY"
-CUSTOM = "CUSTOM"
-TOOLCHAIN = "TOOLCHAIN"
-BUILD = "BUILD"
-FILEMANAGEMENT = "FILEMANAGEMENT"
-DEPENDENCIES = "DEPENDENCIES"
-LICENSE = "LICENSE"
-EXTENSIONS = "EXTENSIONS"
-MODULES = "MODULES"
-OTHER = "OTHER"
+HIDDEN = (-1, 'hidden')
+MANDATORY = (0, 'mandatory')
+CUSTOM = (1, 'easyblock-specific')
+TOOLCHAIN = (2, 'toolchain')
+BUILD = (3, 'build')
+FILEMANAGEMENT = (4, 'file-management')
+DEPENDENCIES = (5, 'dependencies')
+LICENSE = (6, 'license')
+EXTENSIONS = (7, 'extensions')
+MODULES = (8, 'modules')
+OTHER = (9, 'other')
 
 ALL_CATEGORIES = {
```

Comment:
ok, i overlooked something :smile: 
i'm just afraid that this very error prone
maybe add a test with
```python
names=['hidden', ...]
for ind,name in enumerate(names):
    constname=name.replace('-').upper()
    # this validates the import
    # i thinkl import returns too
    constvalue=import('easybuild.framework.easyconfig', constname)
    assert(constvalue, ALL_CATEGORIES[constname])    
```
(or however this works again in python :smile:)